packages/ai-mcp/tests/apps/call-handler.test.ts

```
@@ -0,0 +1,240 @@
+// Per-test variable so individual tests can override the exposed tool list.
+let mockToolsList: Array<MockTool> = [{ name: 'place_order' }]
+
+vi.mock('../../src/client', () => ({
+  createMCPClient: vi.fn(async () => ({
+    tools: async () => mockToolsList,
+    callTool: callToolMock,
+    close: closeMock,
+  })),
+}))
+
+import { createMcpAppCallHandler } from '../../src/apps/call-handler'
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)